base/message_loop/message_pump_io_starboard.cc

```diff
@@ -151,9 +205,16 @@ bool MessagePumpIOStarboard::Watch(SbSocket socket,
     SbSocketWaiterRemove(waiter_, old_socket);
   }
 
+  if (!SbSocketIsValid(socket)) {
+    NOTREACHED() << "Invalid socket" << socket;
```

Comment:
nit: add a space at the end of `"Invalid socket"`

Reply:
Done